pkg/shp/cmd/build/list.go

```diff
@@ -0,0 +1,73 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/spf13/cobra"
+
+	"github.com/shipwright-io/cli/pkg/shp/cmd/runner"
+	"github.com/shipwright-io/cli/pkg/shp/params"
+)
+
+// ListCommand struct contains user input to the List subcommand of Build
+type ListCommand struct {
+	cmd *cobra.Command
+
+	noHeader bool
+}
+
+func listCmd() runner.SubCommand {
+	listCommand := &ListCommand{
+		cmd: &cobra.Command{
+			Use:   "list [flags]",
+			Short: "List Builds",
+		},
+	}
+
+	listCommand.cmd.Flags().BoolVar(&listCommand.noHeader, "no-header", false, "Do not show columns header in list output")
+
+	return listCommand
+}
+
+// Cmd returns cobra command object of List subcommand
+func (sc *ListCommand) Cmd() *cobra.Command {
+	return sc.cmd
+}
+
+// Complete fills object with user input data
+func (sc *ListCommand) Complete(params *params.ShipwrightParams, args []string) error {
+	return nil
+}
+
+// Validate checks user input data
+func (sc *ListCommand) Validate() error {
+	return nil
+}
+
+// Run contains main logic of List subcommand of Build
+func (sc *ListCommand) Run(params *params.ShipwrightParams) error {
+	// Initialize tabwriter for command output
+	writer := tabwriter.NewWriter(os.Stdout, 0, 8, 2, '\t', 0)
```

Comment:
I would add a TODO here to support multiple output formats. Hopefully there are kubectl libraries we can take advantage of.

Reply:
done